### PR TITLE
Various Fabric documentation updates

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

See title

cc @borda @carmocca @justusschock @awaelchli

### CODE REVIEW

File: src/lightning_fabric/README.md [hunk 1]
```diff
@@ -22,53 +22,182 @@ ______________________________________________________________________
 
 </div>
 
-## Maximum flexibility, minimum code changes
+# Lightning Fabric: Expert control.
```
Comment (carmocca) [inline src/lightning_fabric/README.md:25]: These changes are copied from the root README

File: docs/source-fabric/index.rst [hunk 2]
```diff
@@ -16,40 +16,36 @@ Fabric is the fast and lightweight way to scale PyTorch models without boilerpla
 
 .. code-block:: diff
 
-      import torch
-      import torch.nn as nn
-      from torch.utils.data import DataLoader, Dataset
+    + import lightning as L
```
Comment (carmocca) [inline docs/source-fabric/index.rst:19]: This is copied from the root README example

**Review by Borda — APPROVED** [review]

File: src/lightning_fabric/README.md (hunk 1 above)
Comment (awaelchli) [reply src/lightning_fabric/README.md:25]: Hmm, I dont feel like this had to be changed here. That Fabric is fully ready for LLM is not correct imo.

Comment (awaelchli) [reply src/lightning_fabric/README.md:25]: I'm fine Fine with it if it is requested

File: src/lightning_fabric/README.md [hunk 3]
```diff
@@ -22,53 +22,182 @@ ______________________________________________________________________
...
+import torch; import torchvision as tv
+
+fabric = L.Fabric()
+fabric.launch()
+
+model = tv.models.resnet18()
+optimizer = torch.optim.SGD(model.parameters(), lr=0.001)
+model, optimizer = fabric.setup(model, optimizer)
+
+dataset = tv.datasets.CIFAR10("data", download=True,
```
Comment (awaelchli) [inline src/lightning_fabric/README.md:88]: Is the formatting intentional like this?

**Review by awaelchli — APPROVED** [review]

File: src/lightning_fabric/README.md (hunk 3 above)
Comment (carmocca) [reply src/lightning_fabric/README.md:88]: Yes. See other comment.

File: src/lightning_fabric/README.md (hunk 1 above)
Comment (carmocca) [reply src/lightning_fabric/README.md:25]: This is just copy pasted verbatim from the root readme. Whatever we have in there, we should have were aswell

**Pull request merged**